lib/cli-engine.js

```diff
@@ -670,15 +674,28 @@ class CLIEngine {
             // replace \ with / for Windows compatibility
             format = format.replace(/\\/g, "/");
 
+            let cwd = this.options ? this.options.cwd : process.cwd();
+
+            const namespace = naming.getNamespaceFromTerm(format);
+
+            if (namespace) {
+                cwd = `${cwd}/${namespace}`;
+                format = naming.removeNamespaceFromTerm(format);
+            }
+
             let formatterPath;
 
             // if there's a slash, then it's a file
             if (format.indexOf("/") > -1) {
-                const cwd = this.options ? this.options.cwd : process.cwd();
-
                 formatterPath = path.resolve(cwd, format);
             } else {
-                formatterPath = `./formatters/${format}`;
+                const npmFormat = naming.addPrefixToTerm("eslint-formatter-", format);
+
+                try {
+                    formatterPath = resolver.resolve(npmFormat, cwd);
+                } catch (_) {
```

Comment:
Nit: Could you change `_` to `e`? We use `e` everywhere in the codebase, just want to be consistent

Reply:
Done